test/database.ts

```diff
@@ -2307,6 +2307,30 @@ describe('Database', () => {
       assert.strictEqual(bounds, fakeTimestampBounds);
     });
 
+    it('should throw error if maxStaleness is passed in the timestamp bounds to the snapshot', () => {
+      const fakeTimestampBounds = {maxStaleness: 10};
```

Comment:
what will happen if someone assigns them with `null/undefined`?

Reply:
Thanks for identifying this. I missed this check earlier.